app/views/discussion_mailer/_footer.html.haml

```diff
@@ -1,12 +1,12 @@
 .thread-mailer__footer
-  - if @info.recipient.is_logged_in?
-    %img.thread-mailer__footer-image{src: @info.pixel_src, alt: '', width: 1, height: 1}
+  - if recipient.is_logged_in?
+    %img.thread-mailer__footer-image{src: pixel_src(event, recipient), alt: '', width: 1, height: 1}
```

Comment:
can we verify that this works?

Reply:
i notice that the specs have been commented out, and we left some comments. let's pick this up separately